hips/utils/tile.py

```diff
@@ -0,0 +1,18 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""HEALpy wrapper functions.
+
+This module contains wrapper functions around HEALPix utilizing
+the healpy library.
+"""
+
+from pathlib import Path
+
+__all__ = [
+    'tile_path',
+]
+
+
+def tile_path() -> Path:
```

Comment:
This is required for the test cases to pass, otherwise I would have to hardcode the path within `tile.py`.